website/docs/cloud-docs/workspaces/dynamic-provider-credentials/vault-configuration.mdx

```diff
@@ -116,22 +116,22 @@ We recommend setting token_ttl to a relatively short value. Terraform Cloud can
-| `TFC_VAULT_ADDR`          | The address of the Vault instance to authenticate against.                      | Will also be used to set `VAULT_ADDR` in the run environment.                                                                                                                                                                           |
-| `TFC_VAULT_RUN_ROLE`      | The name of the Vault role to authenticate against (`tfc-role`, in our example) | Optional if `TFC_VAULT_PLAN_ROLE` and `TFC_VAULT_APPLY_ROLE` are both provided. These variables are described [below](/terraform/cloud-docs/workspaces/dynamic-provider-credentials/vault-configuration#optional-environment-variables) |
+| Variable                  | Value                                                                           | Notes                                                                                                                                                                                                                                                                                         |
+|---------------------------|---------------------------------------------------------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `TFC_VAULT_PROVIDER_AUTH` | `true`                                                                          | Requires **v1.7.0** or later if self-managing agents. Must be present and set to `true`, or Terraform Cloud will not attempt to authenticate to Vault.                                                                                                                                        |
+| `TFC_VAULT_ADDR`          | The address of the Vault instance to authenticate against.                      | Requires **v1.7.0** or later if self-managing agents. Will also be used to set `VAULT_ADDR` in the run environment.                                                                                                                                                                           |
+| `TFC_VAULT_RUN_ROLE`      | The name of the Vault role to authenticate against (`tfc-role`, in our example) | Requires **v1.7.0** or later if self-managing agents. Optional if `TFC_VAULT_PLAN_ROLE` and `TFC_VAULT_APPLY_ROLE` are both provided. These variables are described [below](/terraform/cloud-docs/workspaces/dynamic-provider-credentials/vault-configuration#optional-environment-variables) |
```

Comment:
Non-blocking nitpick: We could add a period here to keep it consistent with the other env var values:
```suggestion
| `TFC_VAULT_RUN_ROLE`      | The name of the Vault role to authenticate against (`tfc-role`, in our example). | Requires **v1.7.0** or later if self-managing agents. Optional if `TFC_VAULT_PLAN_ROLE` and `TFC_VAULT_APPLY_ROLE` are both provided. These variables are described [below](/terraform/cloud-docs/workspaces/dynamic-provider-credentials/vault-configuration#optional-environment-variables) |
```